index.html

```diff
@@ -394,11 +396,23 @@ <h4>Computation steps</h4>
                     <li>For <code>:after</code> pseudo elements, <a class="termref">User agents</a> MUST append <abbr title="Cascading Style Sheets">CSS</abbr>  textual content, without a space, to the textual content of the <code>current node</code>. </li>
                   </ul>
                 </li>
-                <li id="step2F.iii">For each child node of the <code>current node</code>:
+                <li id="step2F.iii">
+                  Determine the <code>displayed child nodes</code> of the <code>current node</code>:
                   <ol>
-                    <li id="step2F.iii.a">Set the <code>current node</code> to the child node.</li>
-                    <li id="step2F.iii.b">Compute the text alternative of the  <code>current node</code> beginning with step 2.  Set the <code>result</code> to that text alternative.</li>
-                    <li id="step2F.iii.c">Append the <code>result</code> to the <code>accumulated text</code>. </li>
+                    <li id="step2F.iii.a">If the <code>current node</code> has an attached [=shadow root=], set the <code>displayed child nodes</code> to be the child nodes of the [=shadow root=].</li>
+                    <li id="step2F.iii.b">Otherwise, if the <code>current node</code> is a [=slot=] with [=slot/assigned nodes=], set the <code>displayed child nodes</code> to be the [=slot/assigned nodes=] of the <code>current node</code>.</li>
+                    <div><details>
+                      <summary>Comment:</summary>
+                      <p>A [=slot=] with [=slot/assigned nodes=] displays those nodes; otherwise it displays its default content, i.e. its child nodes.</p>
```

Comment:
I think "displays those nodes" may be ambiguous here. Which meaning of "displays" and also which nodes?

Reply:
I probably should have said "renders" instead of "displays."

The spec seems to purely speak in terms of the slot's [assigned nodes](https://html.spec.whatwg.org/multipage/scripting.html#the-slot-element), remarking that:

> Returns slot's [assigned nodes](https://dom.spec.whatwg.org/#slot-assigned-nodes), if any, and slot's children otherwise

I can change the language to match this more closely.

Reply:
I think I found [the right spec language](https://html.spec.whatwg.org/multipage/scripting.html#the-slot-element):

> A [`slot`](https://html.spec.whatwg.org/multipage/scripting.html#the-slot-element) element [represents](https://html.spec.whatwg.org/multipage/dom.html#represents) its [assigned nodes](https://dom.spec.whatwg.org/#slot-assigned-nodes), if any, and its contents otherwise.

I can pretty much just put this verbatim into this comment.